lib/Runtime/Types/DictionaryTypeHandler.cpp

```diff
@@ -869,6 +869,81 @@ namespace Js
     }
 
     template <typename T>
+    BOOL DictionaryTypeHandlerBase<T>::DeleteProperty(DynamicObject *instance, JavascriptString *propertyNameString, PropertyOperationFlags propertyOperationFlags)
```

Comment:
Sure, what I mean is it will be good to have those 2 methods next to each other.

---
In reply to: [78817010](https://github.com/Microsoft/ChakraCore/pull/1542#discussion_r78817010) [](ancestors = 78817010,78791124)